.github/workflows/build.yml

```diff
@@ -161,11 +161,6 @@ jobs:
     needs: build-bicep
 
     steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0 # avoid shallow clone so nbgv can do its work.
-          submodules: true
```

Comment:
Won't getting rid of this cause problems? Bicep uses the embedded version info for compilation logic (e.g. [here](https://github.com/Azure/bicep/blob/57854097106ceb4c6f24f6b0d023001841f3c03e/src/Bicep.Core/Emit/TemplateWriter.cs#L921)).

Reply:
Not sure. I split the steps and added a separate one for VS so that it can happen in parallel. I need access to bicep core and language server assemblies as it needs to be included in VS vsix. This step I believe checks out repository. So I need to add/duplicate step to build Bicep.sln in windows configuration. Do you know of a way to avoid duplication?

Reply:
Added a complete new step for bicep in VS. It runs in parallel with the rest of the steps and finishes earlier. So technically doesn't add any extra time to existing build on CI